docs/manual/intro-to-boa.md

```diff
@@ -274,6 +274,42 @@ In Node.js version < `v14.x`, you also need to add the [`--experimental-modules`
 $ node --experimental-modules --experimental-loader @pipcook/boa/esm/loader.mjs app.mjs
 ```
 
+### Python Generators in JS
+
+The package is able to handle the Python generator class in JavaScript directly:
+
+```python
+# Write a Python Generator in count_down.py
+def count_down(count):
+  while count >= 0:
+    yield count
+    count -= 1
+```
+
+```js
+const boa = require('@pipcook/boa')
+const countDown = boa.import('path_to_above_py')
+const generator = countDown(3); // Generator
+
+// You can access data via the following syntax
+for (const element of generator) {
+  console.log(element) // 3 2 1 0
+}
+
+// or use typical next syntax
+
+const generator = countDown(3); // Generator
```

Comment:
I think this example should be the first case, and it's good to split into different blocks.